mars-app-spa/src/NasaCard.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+const NasaCard: React.FC<{title: string, imgSrc: string, paragraph1: string, paragraph2: string}> = (props) => {
+    return(
+        <div>
+            <h1> {props.title} </h1>
+            <img src = {props.imgSrc} />
+            <p> {props.paragraph1} </p>
+            <p> {props.paragraph2} </p>
```

Comment:
Can't remember if it was with you I mentioned that we prefer not to have the whitespace in HTML. Though browsers will generally trim the surrounding whitespace, we'd prefer our HTML to closely reflect what's rendered.